README.md

```diff
@@ -31,3 +31,13 @@ For now only me :).
 ## License
 
 A short snippet describing the license (MIT, Apache, etc.)
+
+
+Questions:
+- jak gdzieś wspomniałem - niby jest spring boot i conifguracja, ale potem nie tworzonę są beany/componenty 
+- niby jest lombok ale nie użyty
+- nie wiem co nam tutaj dają interfejsy - rozumiem jak mamy wiele implementacji ale tutaj czasem mam wrażenie że to za dużo
+- podział na warstwy a nie na domeny - co powoduje że struktura nie jest oczywista
```

Comment:
Zgadzam się. Większość aplikacji jakie piszę to aplikacje webowe(MVC) stąd odruch dążenia do warst, zresztą stąd też odruchowo dodany lombok i spring boot w zadaniu z wyspami :D.